From 80be8aed50307e394fd0cf72948038b7ccccc2ce Mon Sep 17 00:00:00 2001
From: Will Thompson <wjt@endlessos.org>
Date: Mon, 6 Feb 2023 10:26:47 +0000
Subject: [PATCH] metainfo: Add OARS age-rating data

This data is used by software centre applications such as GNOME Software.
The specification and generator are at https://hughsie.github.io/oars/.

The "social-chat" attribute is defined as:

> Online Text-only Messaging
> --------------------------
>
> Defined as any messaging system connected to the Internet.

The 'intense' value (the highest level) is defined as:

> Uncontrolled chat functionality between users.

Gobby is a network-connected text editor so its entire purpose is
indeed uncontrolled text communication between users over a network or
the internet.

At the time of writing, this rating causes libappstream to assign the
app a minimum age of 13 years.
---
 gobby-0.5.metainfo.xml.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gobby-0.5.metainfo.xml.in b/gobby-0.5.metainfo.xml.in
index 1372e4a..d2af2be 100644
--- a/gobby-0.5.metainfo.xml.in
+++ b/gobby-0.5.metainfo.xml.in
@@ -50,4 +50,8 @@
       </description>
     </release>
   </releases>
+
+  <content_rating type="oars-1.1">
+    <content_attribute id="social-chat">intense</content_attribute>
+  </content_rating>
 </component>
-- 
2.30.2

